src/backend_uart/logic/demo/nexys4ddr/README.md

```diff
@@ -22,18 +24,18 @@ If you have not done this before you need to download and build glip:
 
     git clone https://github.com/TUM-LIS/glip.git
     cd glip
-	./autogen.sh
-	mkdir build; cd build
-	../configure
-	make
-	sudo make install
+    ./autogen.sh
+    mkdir build; cd build
+    ../configure --enable-uart
```

Comment:
yes, it is. Nothing changed in this regard, just made it explicit to avoid breaking the docs if we should ever change the defaults.